android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1753,7 +1753,7 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
       } else if (url.indexOf("refreshBannerHeight") >= 0) {
         int start = url.indexOf("change=") + 7;
         String change = url.substring(start);
-        currentBanner = (change.equals("loaded")) ?
+        currentBanner = (change.equals("active")) ?
```

Comment:
hmm, the banner comes up on app start.
But if I go to "Settings" and Disable predictions, I get this

![Screenshot_1587445031](https://user-images.githubusercontent.com/7358010/79826982-55d56380-83c7-11ea-852e-e05da072fb19.png)

Reply:
This should be fixed now; I had to add some extra code to manage this case.